src/content/app/genome-browser/components/browser-cog/BrowserCog.test.tsx

```diff
@@ -21,8 +21,8 @@ import { faker } from '@faker-js/faker';
 
 import BrowserCog, { BrowserCogProps } from './BrowserCog';
 
-jest.mock('../browser-track-config/BrowserTrackConfig', () => () => (
-  <div id="browserTrackConfig">BrowserTrackConfig</div>
+jest.mock('../track-settings-panel/TrackSettingsPanel', () => () => (
+  <div id="trackSettingsPanel">TrackSettingsPanel</div>
```

Comment:
We can use `data-test-id` instead when we need IDs for testing purposes only: https://testing-library.com/docs/queries/bytestid/

Reply:
We can; but we are mocking this component anyway; it doesn't really matter which selector we then use to find it in the rendered result.

data-test-ids are more useful when we want to add a selector to an element in a real component that is used only during a test.